environment.yml

```diff
@@ -0,0 +1,66 @@
+name: cognoma-cancer-data
+dependencies:
```

Comment:
I submitted an issue to anaconda: https://github.com/ContinuumIO/anaconda-issues/issues/925. Let's see how that goes.